code/modules/admin/fun_balloon.dm

```diff
@@ -91,16 +91,30 @@
 	icon = 'icons/obj/items_and_weapons.dmi'
 	icon_state = "syndballoon"
 	anchored = TRUE
+	var/shuttle_crash = TRUE //this determines if the shuttle will crash into the station.
```

Comment:
Why have this var at all? Why would you put this effect on a shuttle, only to disable the effect?